src/bioutils/normalize.py

```diff
@@ -31,12 +32,18 @@ def normalize(
     sequence,
     interval,
     alleles,
-    mode=NormalizationMode.EXPAND,
+    mode: Optional[NormalizationMode] = NormalizationMode.EXPAND,
     bounds=None,
     anchor_length=0,
+    trim: bool = True,
 ):
     """Normalizes the alleles that co-occur on sequence at interval, ensuring comparable representations.
 
+    Normalization performs three operations:
+    - trimming
+    - shuffling
+    - anchoring
```

Comment:
is there any point in moving the three operations from below into three sub-methods?

Reply:
I agree, but the slightly monolithic state of this code was a preexisting condition. We can address that later.